std_msgs/String name
uint8 type 
std_msgs/String value

uint8 TYPE_NA             = 0
uint8 TYPE_INT            = 1
uint8 TYPE_INT_ARRAY      = 101
uint8 TYPE_UINT           = 2
uint8 TYPE_UINT_ARRAY     = 102
uint8 TYPE_FLOAT          = 3
uint8 TYPE_FLOAT_ARRAY    = 103
uint8 TYPE_STRING         = 4
uint8 TYPE_STRING_ARRAY   = 104
